frontend/src/pages/Gallery.jsx

```diff
@@ -26,12 +28,42 @@ function Gallery() {
         setGalleryData(testGalleryData);
     }, []);
 
+    // Function to handle new image creation
+    const handleCreateNewImage = (newImage) => {
```

Comment:
The submission of create image is supposed to create a new canvas on the **EDITOR** page. Not the gallery page. It will be activated by a button on the toolbar or context menu of the editor.